packages/google-pay-integration/src/google-pay-paypal-commerce/google-pay-paypal-commerce-payment-strategy.ts

```diff
@@ -77,8 +77,19 @@ export default class GooglePayPaypalCommercePaymentStrategy extends GooglePayPay
     }
 
     private async confirmOrder(orderId: string, confirmOrderData: ConfirmOrderData) {
-        const googlePayPayPalSdk =
-            await this._paypalCommerceScriptLoader.getPayPalGooglePaySdkOrThrow();
+        const state = this._paymentIntegrationService.getState();
+        const paymentMethod =
+            state.getPaymentMethodOrThrow<GooglePayPayPalCommerceInitializationData>(
+                this._getMethodId(),
+            );
+
+        const currencyCode = state.getCartOrThrow().currency.code;
+
+        const googlePayPayPalSdk = await this._paypalCommerceScriptLoader.getPayPalSDK(
```

Comment:
It might be better to rename this variable as paypalSdk to be clear what returns getPayPalSDK method

Reply:
renamed, thanks!